extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -20,11 +20,13 @@ const TrackIcon = ( { isPlaying, isError, className } ) => {
 
 	if ( isError ) {
 		name = 'error';
-		/* translators: This is text to describe the current state. This will go before the track title, such as "Error: [The title of the track]" */
+		/* translators: This is text to describe the current state. This will go
+		before the track title, such as "Error: [The title of the track]". */
 		hiddenText = __( 'Error:', 'jetpack' );
 	} else if ( isPlaying ) {
 		name = 'playing';
-		/* translators: Text to describe the current state. This will go before the track title, such as "Playing: [The title of the track]" */
+		/* translators: Text to describe the current state. This will go before the
+		track title, such as "Playing: [The title of the track]". */
```

Comment:
```suggestion
		/*
		 * translators: Text to describe the current state. This will go before the
		 * track title, such as "Playing: [The title of the track]".
		 */
```

Reply:
This won't be picked up (see https://github.com/Automattic/jetpack/pull/15281#discussion_r404743899)